apps/dashboard/app/controllers/module_browser_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ModuleBrowserController < ApplicationController
+  def index
+    # FIXME: select clusters from Configuration.job_clusters
+    @clusters = OodAppkit.clusters.select(&:job_allow?).reject { |c| c.metadata.hidden }
+    @selected_clusters = params[:cluster]
```

Comment:
I doubt we'll want to use query params. I'd say let's show everything in the page and have Javascript/bootstrap tabs hide and show everything.